scripts/setup-triggers.sh

```
@@ -0,0 +1,32 @@
+
+# Exit on error
+set -e
+
+# Check if .env exists
+if [ ! -f .env ]; then
+  echo "Error: .env file not found"
+  exit 1
+fi
+
+# Load .env
+export $(cat .env | grep -v '^#' | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)